openedx/core/djangoapps/credit/views.py

```diff
@@ -29,6 +29,59 @@
 log = logging.getLogger(__name__)
 
 
+@require_GET
+def get_provider_detail(request, provider_id):
+    """
+    get the details of required provider
+
+    This end-point will get the details of providers against
+    given parameter provider_id.
+
+    **Example Usage:**
+
+        Get /api/credit/v1/provider/hogwarts
```

Comment:
This is the preferred URL format; however, it doesn't match with the real URL format.